Source/AliveLibAO/Abe.cpp

```diff
@@ -1614,8 +1614,8 @@ void Abe::sub_422FC0()
     }
 
     // TODO: Check mask is correct
-    const int mask = field_BC_sprite_scale != FP_FromInteger(0) ? 1 : 0x10;
-    if (field_F4_pLine && (mask & (1 << field_F4_pLine->field_8_type)))
+    const int mask = field_BC_sprite_scale != FP_FromDouble(0.5) ? 0x11 : 0x10;
```

Comment:
I think this should be 0x01 and 0x10 ya know, it would match what AE does as well...

Reply:
It should, but it isn't what IDA says...

AE doesn't have the (1 << field_F4_pLine->field_8_type) part too